### DISCUSSION TITLE
Background Cycle Variable

### DISCUSSION THREAD

**Discussion by nevermore23274** [JaKooLit/Fedora-Hyprland#28] [General]
So I've been using these dots for a bit now, and they're great! However, I can't figure out how to stop the backgrounds from cycling. (or how to control the frequency at which they do so) I think it's every 30 minutes or so that it changes?

Could someone help point me in the right direction?

**Comment by JaKooLit** [comment]
https://github.com/JaKooLit/Hyprland-Dots/wiki/FAQ#%EF%B8%8F-automatic-wallpaper-change
